src/app/sell/handlres/put.handler.ts

```diff
@@ -0,0 +1,21 @@
+import {RequestHandler, Request, Response} from 'express';
+import { updateSell } from '../sell.service';
+import { ParamsDictionary } from 'express-serve-static-core';
+import { Sell } from '../sell.type';
+import { SellService } from '../sell.service.v2';
+import { dbConn } from '../../app.database';
+
+type Query = {};
+type Body = Sell;
+
+type Req = Request<ParamsDictionary, {}, Body, Query>;
+type Res = Response;
+
+const sellService = new SellService(dbConn);
```

Comment:
If you create this variable here, you will be working with a singleton. This is not a good idea, because the state can change and affect the behavior of the instance.